server/util/muster-utils.ts

```diff
@@ -92,67 +92,74 @@ export async function getRosterMusterStats(args: {
-    const phoneKey = Object.keys(e.customColumns)
+    const phoneKey = Object.keys(entry.customColumns)
       .find(key => key.toLowerCase().indexOf('phone') !== -1);
 
-    if (phoneKey && edipiToPhone[e.edipi] == null) {
-      edipiToPhone[e.edipi] = e.customColumns[phoneKey] as string;
+    if (phoneKey && edipiToPhone[entry.edipi] == null) {
+      edipiToPhone[entry.edipi] = entry.customColumns[phoneKey] as string;
     }
   });
 
   const allowedRosterColumns = await Roster.getAllowedColumns(org, userRole.role);
 
   // Collect reports and reports missed.
-  const individualStats: IndividualStats = {};
-
-  const aggs = musterAggsResponse.aggregations as IndividualsMusterAggregations | undefined;
+  const aggs = musterAggsResponse.aggregations as MusterRosterAggregations | undefined;
   const buckets = aggs?.muster.buckets ?? [];
   for (const bucket of buckets) {
     const { edipi, reported } = bucket.key;
 
-    if (!individualStats[edipi]) {
-      individualStats[edipi] = {
+    if (!rosterStats[edipi]) {
+      rosterStats[edipi] = {
+        totalMusters: 0,
         mustersReported: 0,
-        mustersNotReported: 0,
-        nonMusterPercent: 0,
+        musterPercent: 0,
       };
     }
 
+    rosterStats[edipi].totalMusters += bucket.doc_count;
+
     if (reported) {
-      individualStats[edipi].mustersReported = bucket.doc_count;
-    } else {
-      individualStats[edipi].mustersNotReported = bucket.doc_count;
+      rosterStats[edipi].mustersReported = bucket.doc_count;
     }
   }
 
-  // Calculate non-muster percents.
-  for (const edipi of Object.keys(individualStats)) {
-    const data = individualStats[edipi];
-    individualStats[edipi].nonMusterPercent = calcNonMusterPercent(data.mustersReported, data.mustersNotReported);
+  // Calculate muster percents.
+  for (const edipi of Object.keys(rosterStats)) {
+    const data = rosterStats[edipi];
+    rosterStats[edipi].musterPercent = calcMusterPercent(data.totalMusters, data.mustersReported);
   }
 
-  // Build a sorted array of the individuals' stats merged with their roster data.
-  const individuals = Object.keys(individualStats)
+  // Return a sorted array of the individuals' stats merged with their roster data.
+  return Object.keys(rosterStats)
     .filter(edipi => {
-      return (
-        rosterEntriesByEdipi[edipi]
-        && individualStats[edipi].nonMusterPercent > 0
-      );
+      return (rosterEntriesByEdipi[edipi] != null);
     })
     .sort((edipiA, edipiB) => {
       const entryA = rosterEntriesByEdipi[edipiA];
       const entryB = rosterEntriesByEdipi[edipiB];
-      const individualA = individualStats[edipiA];
-      const individualB = individualStats[edipiB];
+      const individualA = rosterStats[edipiA];
+      const individualB = rosterStats[edipiB];
 
-      let diff = individualB.nonMusterPercent - individualA.nonMusterPercent;
+      let diff = individualA.musterPercent - individualB.musterPercent;
       if (diff === 0) {
```

Comment:
Should we maybe do an epsilon style comparison here. These are both floating point values and could potentially equal down to 0.000001%, for example. I guess, we could also round them first and compare the rounded values? Either way, I could see a case in the UI where you'd expect them to be alphabetically ordered (because the numerical values both show the same percentage, but they may not be ... Maybe something like 
`if (Math.abs(diff) < 0.001) {` or something?

Reply:
Good catch.